services/userlog/README.md

```diff
@@ -46,7 +50,7 @@ Deprovision messages announce a deprovision text including a deprovision date of
 
 To delete events for an user, use a `DELETE` request to `ocs/v2.php/apps/notifications/api/v1/notifications` containing the IDs to delete.
 
-Only users with the `admin` role can send a `DELETE` request to the `ocs/v2.php/apps/notifications/api/v1/notifications/global` endpoint to remove a global message, see the [Posting](#posting) section for more details.)
+Sending a `DELETE` request to the `ocs/v2.php/apps/notifications/api/v1/notifications/global` endpoint to remove a global message is a restricted action, see the [Authentication](###authentication) section for more details.)
```

Comment:
why `###authentication` with three `###` and not one `#` to link to the section?

Reply:
Maybe because I don't understand how links in markdown work :smile: I want to link to this section: https://github.com/owncloud/ocis/pull/6946/files#diff-9f0e023d863fd491e153ed61bcc77233353293d76d00de308f2121c47058b3daR41